beacon-chain/rpc/beaconv1/config_test.go

```diff
@@ -169,6 +172,10 @@ func TestGetSpec(t *testing.T) {
 			assert.Equal(t, "23", v)
 		case "genesis_fork_version":
 			assert.Equal(t, "0x47656e65736973466f726b56657273696f6e", v)
+		case "altair_fork_version":
+			assert.Equal(t, "0x"+hex.EncodeToString([]byte("AltairForkVersion")), v)
```

Comment:
Maybe also change the assert of `genesis_fork_version` to this more readable version?